build/helper/metadata_add_all.py

```diff
@@ -51,12 +51,8 @@ def _add_ctypes_variable_name(parameter):
 def _add_ctypes_type(parameter):
     '''Adds a ctypes_type key/value pair to the parameter metadata for calling into the library'''
     parameter['ctypes_type'] = parameter['type']
-    if parameter['direction'] == 'out':
-        if parameter['type'] == 'ViString' or parameter['type'] == 'ViRsrc' or parameter['type'] == 'ViConstString':
-            # These are defined as c_char_p which is already a pointer!
-            parameter['ctypes_type_library_call'] = parameter['ctypes_type']
-        else:
-            parameter['ctypes_type_library_call'] = "ctypes.POINTER(" + parameter['ctypes_type'] + ")"
+    if parameter['direction'] == 'out' or parameter['is_buffer'] is True:
```

Comment:
remove "is True" (pythonic)

Reply:
"[As a rule of thumb, you should always use is with the built-in constants True, False and None](https://stackoverflow.com/questions/27276610/boolean-identity-true-vs-is-true)"

Reply:
Good to know!